### PR TITLE
Fixes in Molecule tests, Wazuh components installation and Versioning Control

### PR DESCRIPTION
**Pull request by rshad**
Hi team!

This PR includes many fixes and issues resolving for Molecule tests, Ansible roles and other components in **wazuh-ansible** repository.

1- **Molecule Tests**: 
Molecule tests, found in the branch [3.9.2_7.1.1_tests](https://github.com/wazuh/wazuh-ansible/tree/3.9.2_7.1.1_tests) were not working as expected. Many errors from different types in different roles. Errors like:

- Ansible Lint
- Pure Installation Errors.
- Idempotence Test errors.
- MAX LOCKED MEMORY limits issue.

1.1 - **_Fixed Ansible-Lint errors._** => Trailing spaces, changing shell directive by command, etc ...

1.2 - **_Fixed installation issues for many roles_**, but dedicated more than half the time to Elasticsearch issues which can be resumed as follows:

- - Issues in Centos x.x < 7.0 and Ubuntu Trusty related to the need for having `Java` installed. However, Elasticsearch 7.* indicates that there is no need to install Java explicitly because it already includes a JVM in its installation. But this way was not the case in CentOS 6 and Ubuntu Trusty.  [**Check here the official documentation**](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html)

```
Elasticsearch is built using Java, and includes a bundled version of OpenJDK from the JDK maintainers (GPLv2+CE) within each distribution. The bundled JVM is the recommended JVM and is located within the jdk directory of the Elasticsearch home directory.
``` 

1.3 - **_Idempotence Test errors:_**
We adapted the failing tasks in order to idempotent. so a task which already made a change in the system, when the idempotence test is run, this change must not be produced again.

1.4 - **_MAX LOCKED MEMORY limits issue._**

When running `Molecule` tests on containers for all distribution, **sometimes** "not always", an error was produced in Ubuntu Trusty when trying to start Elasticsearch:
```
/etc/init.d/elasticsearch: line 122: ulimit: max locked memory: cannot modify limit ... Operation not permitted. ...
```
After investigating, we discovered that this issue is related to the privileges of the user running the corresponding task in Ubuntu Trusty, in such case the user needed more privileges than the ones given by default. in order to solve this issue, we needed to add the directive `privileged: true` for Ubuntu Trusty.
```
  - name: trusty
    image: ubuntu:trusty
    privileged: true
    memory_reservation: 2048m
    ulimits:
      - nofile:262144:262144
```

With this error message, we also faced warnings such as: 
```
fatal: [trusty]: FAILED! => {"changed": false, "msg": "OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.\n"
```
by which the execution failed. **CHECK A POSSIBLE SOLUTION** [**HERE**](https://github.com/elastic/elasticsearch/issues/36828#issuecomment-448564460)

- **Note**: We opened a new [issue](https://github.com/wazuh/wazuh-ansible/issues/205) to follow up this problem.

.2. **Versioning Control for Wazuh components when installing with Ansible**
In this issue  `https://github.com/wazuh/wazuh-ansible/issues/197` we had a request to add versioning for Wazuh Manager, Agent, and Filebeat. This functionality was added successfully for each Wazuh component. So now, we can indicate a determined version of  Wazuh or Elasticsearch when running wazuh-ansible installation.

.3. **Create Ansible playbook in order to automate the uninstallation of already installed Wazuh components**

We created a new role in order to uninstall wazuh-manager.

**Related Branches**
- https://github.com/wazuh/wazuh-ansible/tree/fix_tests
- https://github.com/wazuh/wazuh-ansible/tree/3.9.2_7.1.1_tests
- https://github.com/wazuh/wazuh-ansible/tree/issue-198

**Related Issues**
- https://github.com/wazuh/wazuh-ansible/issues/100
- https://github.com/wazuh/wazuh-ansible/issues/59
- https://github.com/wazuh/wazuh-ansible/issues/198
- https://github.com/wazuh/wazuh-ansible/issues/197
- https://github.com/wazuh/wazuh-ansible/issues/203

Kind Regards,

Rshad Zhran

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added 
 
-- Update to Wazuh v3.9.2
+- Update to Wazuh  v3.9.2
```
Comment (manuasir) [inline CHANGELOG.md]: Please remove the additional space here.

File: molecule/elasticsearch/molecule.yml [hunk 2]
```diff
@@ -48,7 +52,7 @@ provisioner:
     ANSIBLE_ROLES_PATH: ../../roles
   lint:
     name: ansible-lint
-    enabled: true
+    enabled: false
```
Comment (manuasir) [inline molecule/elasticsearch/molecule.yml]: This should be set as `true` please.

File: molecule/external_packages/jdk-8u211-linux-x64.rpm [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<html>
```
Comment (manuasir) [inline molecule/external_packages/jdk-8u211-linux-x64.rpm]: Remove this file.

File: molecule/filebeat/molecule.yml [hunk 4]
```diff
@@ -5,6 +5,9 @@ driver:
   name: docker
 lint:
   name: yamllint
+  options:
```
Comment (manuasir) [inline molecule/filebeat/molecule.yml]: Why is this necessary?

Comment (rshad) [reply molecule/filebeat/molecule.yml]: Hi Manuel,

This is needed because `yamllint` testing fails in virtual environments. 

Check this [issue](https://github.com/ansible/molecule/issues/1466#issuecomment-462476230) for more info.

Kind Regards,

Rshad

Comment (manuasir) [reply molecule/filebeat/molecule.yml]: Thanks

File: molecule/default/tests/test_default.py [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 def get_wazuh_version():
     """This return the version of Wazuh."""
-    return "3.9.2"
+    return "3.9.0"
```
Comment (manuasir) [inline molecule/default/tests/test_default.py]: This should return `3.9.2` instead `3.9.0`
